site/content/_future.txt

```diff
@@ -1,3 +1,32 @@
+<<<<<<< HEAD
```

Comment:
Hi, @barrychandler -

Please take a look at this in your editor - you're introducing a lot of unneeded additions which will break this part of the site.